custom/icds_reports/utils/data_pull_scripts/no_vaccine_data.sql

```diff
@@ -0,0 +1,107 @@
+SELECT
```

Comment:
This query serves this:
count of children `0-6 months`, segregated in `0-1m, 1-3m and 3-6m` of buckets who have **never** got the following vaccines. the child count is as of `Nov`.  April to Nov filter is applied because for kid of 0-6m his vaccine can't be given before April.

About Cost:
This looks similar to AP data pull we do. I tried segregated three buckets of children age into three queries but that did not reduced the cost much. SO left in one query.

@calellowitz  @mkangia  fyi.